helpers/util.py

```diff
@@ -58,5 +58,11 @@ def get_used_memory():
     return used_memory
 
 
+def numa_nodes_count():
+    cmd = f"lscpu | grep -oP 'NUMA node\(s\):\s*\K\d+'"
+    out = remote.client.run_cmd(cmd)
```

Comment:
why did you use `remote.client`? I thought we should count numa nodes for VM with TempestaFW